Dockerfile

```diff
@@ -34,3 +34,8 @@ RUN cargo kcov --print-install-kcov-sh | sh
 RUN apt-get -y install python3.6
 RUN apt-get -y install python3-pip
 RUN pip3 install pytest
+
+# Install git.
+RUN apt-get -y install git
+# Install critcmp.
+RUN cargo install critcmp
```

Comment:
It is unrelated, but before publishing a new container version, i would also like us to install `cargo audit` so that we can also fix: https://github.com/rust-vmm/rust-vmm-ci/issues/8

Can you add it as well? I can also add it as an extra commit to your branch, so that we can just publish one new version.

Reply:
Maybe we can also do something along the lines of #26 (but bump things to 1.44.1) for the new version.

Reply:
Sorry for hijacking the PR.

Reply:
Happy to merge this with #26 and add `cargo audit` and `cargo license` on top.

Reply:
Done.

Reply:
1.45 is out :D

Reply:
Agreed offline with @andreeaflorescu to go with 1.44.1 instead as it's been stable for longer.